acceptance/tests/environment/environment_scenario-non_existent.rb

```diff
@@ -46,9 +46,9 @@
   },
   :puppet_agent => {
     :exit_code => 1,
-    :matches => [%r{Warning.*404.*Could not find environment '#{env}'},
+    :matches => [%r{(Warning|Error).*(404|400).*Could not find environment '#{env}'},
```

Comment:
I confirmed that both tests (*-bad* and *-non-existent*) produce 400 error.  in the "bad" test, the environmentpath does not exist on the master (nor agent).  in the non-existent test, the environmentpath exists on the master but the environment itself does not.